tests/test_request_attrs.py

```diff
@@ -690,13 +690,13 @@ def test_date_missing(self, asgi, attr):
         assert getattr(req, attr) is None
 
     @pytest.mark.parametrize('name,value,attr,default', [
-        ('Accept', 'x-falcon', 'accept', '*/*'),
-        ('Authorization', 'HMAC_SHA1 c590afa9bb59191ffab30f223791e82d3fd3e3af', 'auth', None),
-        ('Content-Type', 'text/plain', 'content_type', None),
-        ('Expect', '100-continue', 'expect', None),
-        ('If-Range', 'Wed, 21 Oct 2015 07:28:00 GMT', 'if_range', None),
-        ('User-Agent', 'testing/3.0', 'user_agent', None),
-        ('Referer', 'https://www.google.com/', 'referer', None),
+        # ('Accept', 'x-falcon', 'accept', '*/*'),
+        # ('Authorization', 'HMAC_SHA1 c590afa9bb59191ffab30f223791e82d3fd3e3af', 'auth', None),
+        # ('Content-Type', 'text/plain', 'content_type', None),
+        # ('Expect', '100-continue', 'expect', None),
+        # ('If-Range', 'Wed, 21 Oct 2015 07:28:00 GMT', 'if_range', None),
+        ('User-Agent', 'testing/3.0', 'user_agent', 'falcon-client/' + falcon.__version__),
+        # ('Referer', 'https://www.google.com/', 'referer', None),
```

Comment:
The comments here seem a debug leftover

Reply:
I've uncommented those fixtures. Let's see what happens :boom: